GG/GG/utf8/checked.h

```diff
@@ -120,13 +120,13 @@ namespace utf8
         }
         else if (cp < 0x10000) {              // three octets
             *(result++) = static_cast<uint8_t>((cp >> 12)           | 0xe0);
-            *(result++) = static_cast<uint8_t>((cp >> 6) & 0x3f     | 0x80);
+            *(result++) = static_cast<uint8_t>(((cp >> 6) & 0x3f)   | 0x80);
```

Comment:
Are you sure these changes doesn't break anything? I don't know, but I'd have avoided messing with stuff like this...